bgpd/bgp_vty.c

```diff
@@ -16313,6 +16328,39 @@ DEFUN(no_neighbor_tcp_mss, no_neighbor_tcp_mss_cmd,
 	return peer_tcp_mss_vty(vty, argv[peer_index]->arg, NULL);
 }
 
+DEFUN(bgp_retain_route_target, bgp_retain_route_target_cmd,
+      "[no] bgp retain route-target all",
```

Comment:
Please use DEFPY instead of DEFUN here. Add Also  please update the  documentation for this new command.

Reply:
done. ok.